creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -842,39 +855,51 @@ class SnapbackSM {
    * - Processes all users in chunks
    * - For every user on an unhealthy replica, issues an updateReplicaSet op to cycle them off
    * - For every (primary) user on a healthy secondary replica, issues SyncRequest op to secondary
-   *
-   * @note refer to git history for reference to `processStateMachineOperationOld()`
    */
   async processStateMachineOperation() {
+    /**
+     * Ensure lock available, and acquire it - else error
+     * @notice Bull will not enqueue further jobs if all active spots are full, but this provides added
```

Comment:
wait so this isn't necessary since bull has it? is this just extra protection in case bull doesn't work?

Reply:
exactly, we could prob remove it tbh, with concurrency of 1 it **should** never be used, if concurrency is ever increased this would actually prevent that from happening (which is what we want)

i'm game to remove it i think

Reply:
decided to leave in